ci_framework/playbooks/ceph.yml

```diff
@@ -159,6 +161,22 @@
       loop: "{{ lookup('file', cifmw_cephadm_spec_ansible_host) | from_yaml_all | list }}"
       delegate_to: localhost
 
+    - name: Get already assigned IP addresses
```

Comment:
@fultonj this is the current flow to reserve an IP address:

- Build the list of already assigned `ips` looping over the `edpm` group (I don't need hosts, hence a list containing  `host_to_ip.values()` is enough in this context)
- set the `cifmw_cepahdm_vip` as an empty string
- call the `check_vip` tasks from the cephadm role and filter against the `ips` list

The output of this flow is an available `vip` that can be propagated to the `radosgw` tasks

Reply:
That's good. Hopefully we can just get the original values to start with from the results of the previous task.